### PR TITLE
Add support for tuc

### LINKED ISSUE #2613: [Icon Request] Support for tuc
- [x] I'm sure this issue is not a _duplicate_.
- [x] I'm submitting an icon request:
  - Type: `extension, language id`
  - Icon Name: `tuc`
  - Sample original Icon: logo
  - Extensions: `tuc`
  - Language ids: `TypedUseCase`

- Language Extensions: https://marketplace.visualstudio.com/items?itemName=TypedUseCase.tuc
- More info:
  - [tuc-console](https://github.com/TypedUseCase/tuc-console) (tuc language compiler/parser, examples, ...)
  - [documentation](https://typedusecase.github.io/)
  - [vs code extension](https://github.com/TypedUseCase/tuc-extension)

_Edit: I've moved all tuc repos to the https://github.com/TypedUseCase organization._

### PR DESCRIPTION
**Pull request by MortalFlesh**
_**Fixes #2613**_

**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

### CODE REVIEW

**Comment by KingDarBoja** [comment]
The icon need some changes as the margins are not present.

**Comment by MortalFlesh** [comment]
I edited the svg manually, to "match" other icons. This is originally exported svg from illustrator https://raw.githubusercontent.com/MortalFlesh/tuc-console/master/docs/assets/tuc-logo-32.svg is it better?

I'm not that familiar with svg, so could you add more info about whats wrong, please?

Thank you

**Comment by KingDarBoja** [comment]
@MortalFlesh Please take a look at https://github.com/vscode-icons/vscode-icons/wiki/Contributing

TLDR; the icon has the correct size (32x32px) but lacking the margins on every side (2px per side).

**Comment by MortalFlesh** [comment]
Hello @KingDarBoja 

thanks for clarification - I missed it before.

Is it correct now?

**Comment by MortalFlesh** [comment]
And I can't see why those security checks fail.

**Comment by KingDarBoja** [comment]
I have no idea what happened but looks like the PR got closed and master branch (of MortalFlesh repo) got reset?

**Comment by MortalFlesh** [comment]
Well I have no idea either, I didn't do anything about it.. 

Thanks for reopening the PR by yourself.
